### PR TITLE
Fixes mm-1391 adding travis CI

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

**Comment by coreyhulen** [comment]
This should also fix a large portion of the unit test gremlins.

File: config/config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "LogSettings": {
         "ConsoleEnable": true,
-        "ConsoleLevel": "INFO",
+        "ConsoleLevel": "DEBUG",
```
Comment (jwilander) [inline config/config.json]: Is this meant to be included?

**Comment by jwilander** [comment]
Once my one comment is resolved this is ready for merge

File: config/config.json (hunk 1 above)
Comment (coreyhulen) [inline config/config.json]: yup, intensional so the travis build system captures the error.

**Pull request merged**